fuels-contract/src/constants.rs

```diff
@@ -0,0 +1,3 @@
+pub const GAS_LIMIT: u64 = 1_000_000;
```

Comment:
We should probably put `NATIVE_ASSET_ID` in this file since we're here

Reply:
We could also put this inside: 
`NULL_CONTRACT_ID: &str = "0x00…"` so as to avoid the `null_contract_id` function inside the harness.

Reply:
Yup, might as well move all relevant constants here.